CODEOWNERS

```diff
@@ -101,6 +101,7 @@
 /boards/atmel/                            @nandojve
 /boards/arm/scobc_module1/                @yashi
 /boards/arm/v2m_beetle/                   @fvincenzo
+/boards/arm/verdin_imx8mm_m4/             @hiagofranco
```

Comment:
I don't think we should add to the codeowners file here, the rest of the file paths are out of date for boards and it seems like we may not have migrated them to HWMv2 (@nordicjm is there a reason we chose not to?) I know we are migrating to `MAINTAINERS.yml`, so perhaps that is why

Reply:
Didn't notice that codeowners was out of data, will need to have a look at that. @nashif for comment on use of codeowners file

Reply:
@nashif any update on this point? Should we remove the entry in the CODEOWNERS file?

Reply:
I have made these changes.